docs/KnownIssues.md

```diff
@@ -1,90 +0,0 @@
----
-id: known-issues
-title: Known Issues
-layout: docs
-category: Guides
-permalink: docs/known-issues.html
-next: performance
----
-
-### Devtools "React" Tab Does Not Work
-
-It's [currently not possible](https://github.com/facebook/react-devtools/issues/229) to use the "React" tab in the devtools to inspect app widgets. This is due to a change in how the application scripts are evaluated in the devtools plugin; they are now run inside a Web Worker, and the plugin is unaware of this and so unable to communicate properly with React Native.
-
-However, you can still use the Console feature of the devtools, and debugging JavaScript with breakpoints works too. To use the console, make sure to select the `⚙debuggerWorker.js` entry in the devtools dropdown that by default is set to `<top frame>`.
-
-### Missing Android Modules and Views
-
-The work on React Native for Android started later than React Native for iOS. Most views and modules are now available on Android, with the following exceptions:
-
-#### Views
```

Comment:
I dropped some of these as it did not seem like it was worth calling out in the docs. Many seemed like, if they are still an issue, they would be easily discovered by performing a search for the issue.

Happy to reinstate them, it just seems like there's a grab bag of random issues in this doc that may no longer be relevant.